provider/go.mod

```diff
@@ -270,5 +271,5 @@ require (
 replace (
 	github.com/hashicorp/go-getter => github.com/hashicorp/go-getter v1.4.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20220725190814-23001ad6ec03
-	github.com/hashicorp/terraform-provider-vault => github.com/pulumi/terraform-provider-vault v1.9.1-0.20221013032412-e762e637392b
+	github.com/hashicorp/terraform-provider-vault => github.com/pulumi/terraform-provider-vault v1.9.1-0.20230123235526-378e13675426
```

Comment:
> Question from [pulumi/pulumi#12300](https://github.com/pulumi/pulumi/issues/12300)
> 
> Are we supposed to bump this? If not, why?
> 
> cc @iwahbe @guineveresaenger

@dixler This is a forked provider, so the line you mentioned is required but the associated version is ignored. The real version bump is here ☝️.